lib/beaker-hostgenerator/cli.rb

```diff
@@ -116,69 +117,81 @@ def initialize(argv = ARGV.dup)
         opts.on('-h',
                 '--help',
                 'Display command help.') do
-          puts opts
-          exit
+          @options[:help] = opts.to_s
+        end
+
+        opts.on('-v',
+                '--version',
+                'Display beaker-hostgenerator version number.') do
+          @options[:version] = true
         end
       end
 
       optparse.parse!(argv)
+      @layout = argv[0]
+    end
 
-      if @options[:list_supported_values]
-        print_supported_values
-        raise BeakerHostGenerator::Exceptions::SafeEarlyExit
```

Comment:
I think it's a relatively low-risk member of the API. That coupled with the fact that the `0` major version is intended to be unstable should give us the leeway to remove it now if you prefer. I don't have a preference in this case as I did in the past because in the past the API changes had a higher risk of breaking actual API use cases.